verify_release

```diff
@@ -172,15 +201,29 @@ def main() -> int:
                 # This is expected while build is not reproducible
                 finished("ERROR: PyPI artifacts do not match built release")
                 success = False
+            else:
+                finished("PyPI artifacts match the built release")
 
         progress("Downloading release from GitHub")
         if not verify_github_release(build_version, build_dir):
             # This is expected while build is not reproducible
             finished("ERROR: GitHub artifacts do not match built release")
             success = False
-
-        if success:
-            finished("Github and PyPI artifacts match the built release")
+        else:
+            finished("GitHub artifacts match the built release")
+
+        # NOTE: 'gpg' might prompt for password or ask if it should override files...
+        if args.sign:
+            progress("Signing built release with gpg")
+            if success:
+                key_id = None
+                if args.sign is not True:
```

Comment:
when could this be false?

Reply:
oh, it's True, False or keyid...

Reply:
`,/verify_release --sign 8BA69B87D43BE294F23E812089A2AD3C07D962E8`

Reply:
> oh, it's True, False or keyid...

Actually, it is True, None or keyid